framework/src/times/CSVFileTimes.C

```diff
@@ -22,6 +22,8 @@ CSVFileTimes::validParams()
   params.addParam<unsigned int>("time_column_index", 0, "Index for the column with the time");
   // File is loaded on all processes
   params.set<bool>("auto_broadcast") = false;
+  // CSV file set the time sequence during initialization stage
+  params.set<bool>("dynamic_time_sequence") = false;
```

Comment:
you should set this in the InputTimes too, and in ControllableInputTimes back as true